lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -39,6 +43,30 @@ export async function ensureOnboardingPr(
   logger.trace({ config });
   // TODO #7154
   const existingPr = await platform.getBranchPr(config.onboardingBranch!);
+  if (existingPr) {
+    // skip pr-update if branch is conflicted
+    if (
+      await scm.isBranchConflicted(
+        config.defaultBranch!,
+        config.onboardingBranch!
+      )
+    ) {
+      await ensureComment({
+        number: existingPr.number,
+        topic: 'Branch Conflicted',
+        content: emojify(
+          `:warning: This PR has a merge conflict. However, Renovate is unable to automatically fix that due to edits in this branch. Please resolve the merge conflict manually.\n\n`
+        ),
+      });
+      return;
+    } else {
+      await ensureCommentRemoval({
```

Comment:
This will trigger at least one extra API call every run to check if the comment exists or not. I'm not sure it's so bad to just leave that comment in place as this should be a rare occurrence.

Reply:
Use cache to prevent it maybe?

Reply:
I'm not sure you can easily do that, because comments can be changed between runs. I suggest leaving the comment there for now.